docs/learn/learn-async-backing.md

```diff
@@ -200,38 +196,127 @@ blocks in advance to fit more data, Polkadot with asynchronous backing can deliv
+
+Values of zero for both correspond to synchronous backing: `allowed_ancestry_len = 0` means there
+can be only 1 parablock at a time on the conveyor belt, `max_candidate_depth = 0` means a parachain
+block can be built only on the latest relay parent for that parachain. Those two parameters will be
+initially set to 3 and 2, respectively.
+
+## Terminology
+
+### Candidate Receipt
+
+Saying that a parablock has been included in a realy chain parent does not mean that the entire
+parablock is in the relay chain block. Instead, **candidate receipt** consisting of the hash of the
+parablock, state roots and ID info is placed on the parent block on the relay chain. The relay chain
+does not access the entire state of a parachain but only the values that changed during that block
+and the merkelized hashes of the unchanged values.
+
+### Pipelining
+
+Asynchronous backing is a feature that introduces
+[pipelining](https://www.techtarget.com/whatis/definition/pipelining) to the parachain block
+[generation, backing and inclusion](./learn-parachains-protocol.md). It is analogous to the logical
+pipelining of processor instruction in "traditional" architectures, where some instructions may be
+executed before others are complete. Instructions may also be executed in parallel, enabling
+multiple processor parts to work on potentially different instructions simultaneously.
+
+Bundles of state transitions represented as blocks may be processed similarly. In the context of
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}, pipelining aims to increase the
+throughput of the entire network by completing the backing and inclusion steps for different blocks
+at the same time. Asynchronous backing does not just allow for pipelining within a single pipe (or
+core). It lays the foundation for a large number of pipes (~10 cores) to run for the same parachain
+at the same time. In that way we have two distinct new forms of parallel computation: having one
+block backed and one included simultaneously using one core, and having having `n` blocks backed and
+`n` blocks included simultaneously using `n` cores.
+
+### Contextual Execution
+
+Contextual execution refers to the context built by parablock ancestors used by collators to
+generate new parablocks. The difference in execution context between synchronous and asynchronous
+backing is as follows:
+
+In synchronous backing, the entire execution context comes from the latest relay block and most
+recently included parablock from the same parachain.
+
+In asynchronous backing, a baseline execution context is first set up using a parablock's relay
+parent and the most recently included parablock as of that relay parent. The relay parent need not
+be the most recent relay block. Predecessors of the block under construction live in the
+[unincluded segment](#unincluded-segments) placed directly in the runtime of parachain collators
+(i.e. off-chain).
```

Comment:
I was wrong to use the term off-chain here. Ancestor blocks in the unincluded segment aren't stored on the relay chain. But they are stored on-chain in the parachain runtime.
```suggestion
```